lib/engine/game/g_18_uruguay/step/route.rb

```diff
@@ -12,7 +12,7 @@ def setup
           end
 
           def actions(entity)
-            return %w[].freeze if entity.corporation == @game.rptla
+            return [] if entity.corporation == @game.rptla
             return [] if !entity.operator? || @game.route_trains(entity).empty? || !@game.can_run_route?(entity)
             return [] if entity.corporation? && entity.type == :minor
```

Comment:
Are there any corporations with type `:minor` in this game? Do you mean `return [] if entity.minor?`?

Reply:
Updated